### DISCUSSION TITLE
amamba/user-guide/pipeline/podman

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3814] [Announcements]
# amamba/user-guide/pipeline/podman

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/user-guide/pipeline/podman.html

**Comment by muma378** [comment]
> 则jenkins所在的主机需要安装QEMU。

应该是agent所在的容器里

> 如果jenkins所在的主机是macos或者windows，则需要先启动虚拟机才可以使用podman:

貌似在流水线的语境下不需要？
